pkgs/development/python-modules/json-repair/default.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "json-repair";
+  version = "0.26.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "mangiucugna";
+    repo = "json_repair";
+    rev = "refs/tags/${version}";
+    hash = "sha256-s28p7fmrQwZTiWJnTQjjR4vABNOaiW2ngkyfHgOzczQ=";
+  };
+
+  build-system = [ setuptools ];
+
```

Comment:
Could you add `pythonImportsCheck` for a smoke test?

> pythonImportsCheck is set. This is still a good smoke test even if pytestCheckHook is set.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-contributing